core/backend/src/perftest/TilesGen.test.ts

```diff
@@ -187,7 +187,7 @@ async function generateIModelDbTiles(requestContext: AuthorizedClientRequestCont
 }
 
 describe("TilesGenerationPerformance", () => {
-  const config = require(Config.App.getString("imjs_tile_perf_config")); // eslint-disable-line @typescript-eslint/no-var-requires
+  const config = require(process.env.imjs_tile_perf_config ?? ""); // eslint-disable-line @typescript-eslint/no-var-requires

```

Comment:
Can you make this throw if it's not properly set?

Reply:
Sure. Is this decent enough or is there a more elegant (or 1 line) solution to do this?
```
if (process.env.imjs_tile_perf_config === undefined)
    throw new Error("Could not find imjs_tile_perf_config")
const config = require(process.env.imjs_tile_perf_config);
```

Reply:
that's easy enough to read so lets go with it. Technically you could do it on one line like this,

```
const config = undefined !== process.env.imjs_tile_perf_config ? require(process.env.imjs_tile_perf_config : throw new Error("Could not find imjs_tile_perf_config");
```